### PR TITLE
Phase21 test PR

### PR DESCRIPTION
**Pull request by hiletmis**
This PR is related to https://github.com/api3dao/tasks/issues/599

It has been created for testing purposes of auto-update package/repo feature of phase2.1

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "18.x"
+
+      - name: Install dependencies
+        run: yarn --network-timeout 300000
```
Comment (andreogle) [inline .github/workflows/release.yml:26]: I think we should remove the `--network-timeout` if it's not necessary. This should also use `--frozen-lockfile`

```suggestion
        run: yarn install --frozen-lockfile
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@api3/chains",
+  "name": "@phase21/chains",
```
Comment (andreogle) [inline package.json:2]: I'm confused. Why this change to `main`?

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   "license": "MIT",
-  "version": "4.2.0",
+  "version": "0.1.1",
```
Comment (andreogle) [inline package.json:4]: And reverting this to 0.1.1 on `main`?

File: README.md [hunk 4]
```diff
@@ -19,9 +19,9 @@ The single source of truth for chain-related information across API3 projects
 ## 📦 Installation
 
 ```sh
-npm install @api3/chains --save
-yarn add @api3/chains
-pnpm add @api3/chains
+npm install @phase21/chains --save
```
Comment (andreogle) [inline README.md:22]: The changes in `README.md` don't seem right either?

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+      - name: Build package
+        run: yarn build
+
+      - name: Create Release Pull Request or Publish to npm
```
Comment (andreogle) [inline .github/workflows/release.yml:31]: This name is still confusing to me 🤷🏻‍♂️

File: README.md
Comment (andreogle) [inline README.md:1]: If we're going to be changing the release process, can you please document it here too (at the bottom)?

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        uses: changesets/action@master
+        with:
+          publish: yarn release
```
Comment (andreogle) [inline .github/workflows/release.yml:34]: Referring to this comment

> Changeset creates a pr if a new version is submitted

When does this happen? Is it manual or automatic?

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# https://github.com/changesets/action
+name: Release
+
+on:
+  push:
+    branches:
+      - phase21
```
Comment (andreogle) [inline .github/workflows/release.yml:7]: Doesn't this need to change as well?

**Review by andreogle — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (hiletmis) [reply package.json:2]: I will use `phase21` branch to release on `@phase21` organization while the main branch will remain as `@api3`. there may be some conflicts in the process.

File: package.json (hunk 3 above)
Comment (hiletmis) [reply package.json:4]: This package version is for `@phase21/chains` package. You can consider this branch will not be merged any time soon as it will be for test purposes only.

File: .github/workflows/release.yml (hunk 5 above)
Comment (hiletmis) [reply .github/workflows/release.yml:31]: We can think a better name for it

File: README.md
Comment (hiletmis) [reply README.md:1]: Sure thing. I will add it to the ReadMe if its okay?

File: .github/workflows/release.yml (hunk 7 above)
Comment (hiletmis) [reply .github/workflows/release.yml:7]: First I need to do some mock updates to trigger linked updates. so that is why I need to work on separate branch and keep main branch clean

File: package.json (hunk 2 above)
Comment (hiletmis) [reply package.json:2]: Yes. That's correct.

**Comment by bbenligiray** [comment]
What's the status on this?

**Comment by hiletmis** [comment]
The only requirement is `changeset` atm. `renovate` is already implemented on main

**Comment by hiletmis** [comment]
@bbenligiray this PR has too mant try/error commits so implementing `changeset` has been moved to this [PR](https://github.com/api3dao/chains/pull/210) and it is ready for review
